helm/amko/crds/gslbhostrule_def.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: gslbhostrules.amko.vmware.com
+spec:
+  conversion:
+    strategy: None
+  group: amko.vmware.com
+  names:
+    kind: GSLBHostRule
+    listKind: GSLBConfigList
+    plural: gslbhostrules
+    shortNames:
+    - ghr
+    singular: gslbhostrule
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        type: object
+        properties:
+          spec:
+            type: object
+            properties:
+              gsFqdn:
+                description: "FQDN of the GslbService to which this set of rule applies."
+                type: string
+              ttl:
+                description: "Time To Live. Specify in seconds how long to hold a DNS record."
+                type: integer
```

Comment:
Same, Do you think a validation for this is required? max/min

Reply:
Min is required. Added. Thanks.